src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/GCHandleTests.cs

```diff
@@ -66,15 +66,15 @@ public static void Alloc_Type_ReturnsExpected(object value, GCHandleType type)
 
         public static IEnumerable<object[]> InvalidPinnedObject_TestData()
         {
-            yield return new object[] { new object() };
+            if (!PlatformDetection.IsNetNative)
+                yield return new object[] { new object() };
```

Comment:
The blittability depend on internal implementation details. If you try to pin objects that you do not control the field layout for, you will get different results for different implementation details. There are likely hundreds of other types where desktop, CoreCLR and CoreRT will produce different results here.

I have mentioned in the issue that it maybe best to delete this case from the test.